src/features/create-dao/Steps/TreasuryForm.tsx

```diff
@@ -1,24 +1,56 @@
 import { FC, useContext } from 'react';
 
+import * as Yup from 'yup';
 import { Form, Formik } from 'formik';
 
 import { Wizard } from '@zero-tech/zui/components';
 import { CreateDAOFormContext } from '../';
 
 import styles from './TreasuryForm.module.scss';
+import { WrappedInput } from '../../ui/WrappedInput/WrappedInput';
```

Comment:
nit: let's reorder these imports to match zer0's preferred pattern: https://softwaresurgery.slack.com/archives/CQKH46ADB/p1666060086775939.

Reply:
Changed the order